<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardEdit</class>
 <widget class="QDialog" name="BoardEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(254, 200, 255);</string>
  </property>
  <widget class="QPushButton" name="btn_Back">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 border: 2px solid  rgb(155, 144, 200);
 border-radius: 8px;
 padding-left: auto;
    padding-right: auto;
 color: rgb(0, 0, 0);
 background-color: rgb(187, 159, 207);
 font: 600 12pt &quot;Montserrat SemiBold&quot;;
}
QPushButton:hover {
 border: 2px solid rgb(156, 132, 192);
 background-color: rgb(156, 132, 192);
}
QPushButton:focus {
 border: 2px solid rgb(109, 86, 154);

}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	 color: rgb(0, 0, 0);
	font: 800 22pt &quot;Montserrat ExtraBold&quot;;
}</string>
   </property>
   <property name="text">
    <string>Board Edit</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>601</width>
     <height>164</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btn_Owner">
          <property name="styleSheet">
           <string notr="true">QPushButton {
 border: 2px solid  rgb(155, 144, 200);
 border-radius: 8px;
 padding-left: auto;
    padding-right: auto;
 color: rgb(0, 0, 0);
 background-color: rgb(187, 159, 207);
 font: 600 12pt &quot;Montserrat SemiBold&quot;;
}
QPushButton:hover {
 border: 2px solid rgb(156, 132, 192);
 background-color: rgb(156, 132, 192);
}
QPushButton:focus {
 border: 2px solid rgb(109, 86, 154);

}</string>
          </property>
          <property name="text">
           <string>Add Owner</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Name">
          <property name="styleSheet">
           <string notr="true">QPushButton {
 border: 2px solid  rgb(155, 144, 200);
 border-radius: 8px;
 padding-left: auto;
    padding-right: auto;
 color: rgb(0, 0, 0);
 background-color: rgb(187, 159, 207);
 font: 600 12pt &quot;Montserrat SemiBold&quot;;
}
QPushButton:hover {
 border: 2px solid rgb(156, 132, 192);
 background-color: rgb(156, 132, 192);
}
QPushButton:focus {
 border: 2px solid rgb(109, 86, 154);

}</string>
          </property>
          <property name="text">
           <string>Edit Name</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_owner">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border: 2px solid rgb(156, 132, 192);
	border-radius: 8px;
	padding-left: auto;
    padding-right: auto;
	color: rgb(255, 255, 255);
	background-color: rgb(27, 29, 35);
	font: 600 11pt &quot;Montserrat SemiBold&quot;;
}
QLineEdit:hover {
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus {
	border: 2px solid rgb(48, 50, 62);
	background-color: rgb(43, 45, 56);
}


</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>15</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_name">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border: 2px solid rgb(156, 132, 192);
	border-radius: 8px;
	padding-left: auto;
    padding-right: auto;
	color: rgb(255, 255, 255);
	background-color: rgb(27, 29, 35);
	font: 600 11pt &quot;Montserrat SemiBold&quot;;
}
QLineEdit:hover {
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus {
	border: 2px solid rgb(48, 50, 62);
	background-color: rgb(43, 45, 56);
}


</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>15</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_Type">
        <property name="styleSheet">
         <string notr="true">QPushButton {
 border: 2px solid  rgb(155, 144, 200);
 border-radius: 8px;
 padding-left: auto;
    padding-right: auto;
 color: rgb(0, 0, 0);
 background-color: rgb(187, 159, 207);
 font: 600 12pt &quot;Montserrat SemiBold&quot;;
}
QPushButton:hover {
 border: 2px solid rgb(156, 132, 192);
 background-color: rgb(156, 132, 192);
}
QPushButton:focus {
 border: 2px solid rgb(109, 86, 154);

}</string>
        </property>
        <property name="text">
         <string>Change Privacy Type</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton_private">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton {
 color: rgb(0, 0, 0);
	font: 600 12pt &quot;Montserrat SemiBold&quot;;
}
QRadioButton::indicator:checked {
    background-color: rgb(27, 29, 35) ;
	border: 2px solid white;
	border-radius: 6px;
}

QRadioButton::indicator:unchecked {
    background-color: white;
    border: 2px solid white;
	border-radius: 6px;
}</string>
          </property>
          <property name="text">
           <string>Private</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_public">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton {
 color: rgb(0, 0, 0);
	font: 600 12pt &quot;Montserrat SemiBold&quot;;
}
QRadioButton::indicator:checked {
    background-color: rgb(27, 29, 35) ;
	border: 2px solid white;
	border-radius: 6px;
}

QRadioButton::indicator:unchecked {
    background-color: white;
    border: 2px solid white;
	border-radius: 6px;
}
</string>
          </property>
          <property name="text">
           <string>Public</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
